package.json

```diff
@@ -149,6 +149,11 @@
                 "type": "string",
                 "description": "The source root on this machine that is the equivalent to the serverSourceRoot on the server."
               },
+              "pathMappings": {
+                "type": "object",
+                "default": {},
+                "description": "A mapping of multiple localSourceRoots on this machine to remote serverSourceRoots on the server."
```

Comment:
this shouldn't mention the name `localSourceRoot` anymore, and the old settings should be deprecated to not have two settings doing the same thing without clear precedence